ci/acceptance_tests.sh

```diff
@@ -21,6 +21,12 @@ SELECTED_TEST_SUITE=$1
 # BUILD_ID unless you set this magic flag:  https://wiki.jenkins.io/display/JENKINS/ProcessTreeKiller
 export BUILD_ID=dontKillMe
 
+# Always run the halt, even if the test times out or an exit is sent
+cleanup() {
+  bundle exec rake qa:vm:halt
```

Comment:
what is this tasks' behaviour when the VM is already stopped? Will the `set -e` at the head of this script cause us to fail the whole build if the VM has already shut down cleanly?